bioinf-tools-1/add_modules/input_output_module.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+def write_as_dict(input_fastq: str) -> dict:
+    seqs = {}
+    with open(input_fastq) as file:
+        for line in file:
+            if line.startswith("@"):
+                line1 = line.strip()
+                name = line1.split(" ")[0]
+                if not name:
+                    break
+                line2 = file.readline()
+                sequence = line2.strip()
+                line = file.readline()
+                line3 = file.readline()
+                quality = line3.strip()
+                seqs[name] = (sequence, quality)
+    return seqs
+
+
+def save_filtered_sequences(seqs_checked: dict, output_fastq: str) -> None: 
+    if output_fastq is None:
+        output_fastq = "output_fastq.fastq"
+    output_path = os.path.join("filtered", output_fastq)
+    with open(output_fastq, "w") as file:
+        for seq_id, (sequence, quality) in seqs_checked.items():
+            file.write(f"{sequence}\n")
```

Comment:
```suggestion
def save_filtered_sequences(output_fastq: str, name: str, sequence: str, quality: str) -> None:
    if output_fastq is None:
        output_fastq = "output_fastq.fastq"
    output_path = os.path.join("filtered", output_fastq)
    
    with open(output_path, "w") as file:
        for seq_id, (sequence, quality) in seqs_checked.items():
            file.write(f"@{seq_id}\n")
            file.write(f"{sequence}\n")
            file.write("+\n") 
            file.write(f"{quality}\n")
```
Кажется, ты записываешь только последовательность ДНК, а надо все строки по формату. А в with open не используешь ту директорию, которую сделал_а (`output_path`)